pom.xml

```diff
@@ -73,6 +73,11 @@
       <artifactId>ssh-credentials</artifactId>
       <version>1.6.1</version>
     </dependency>
+    <dependency>
```

Comment:
Yup, not released yet. I can remove it for now and just bump the minimum Jenkins version temporarily so that this passes.